### ISSUE TITLE
Shortcut for switching to specific keyboard layout(s)

### ISSUE THREAD

**Issue by tsujan** [lxqt/lxqt-config#447]
When there are multiple layouts, it'll be useful if a shortcut can also be added for switching to each one, especially when languages are (going to be) mixed while editing a document.

**Comment by tsujan** [comment]
`lxqt-config` may not be able to do it; perhaps, it's related to `lxqt-globalkeys`, or the former should change the config of the latter, or something else is needed. All is all, it isn't as simple as it appears.

**Comment by johnvranos** [comment]
Hi, I am test-using LXQT, with Manjaro Linux.

lxqt-config appears to provide the ability of changing keyboard layout with a keyboard shortcut.

**Comment by tsujan** [comment]
> lxqt-config appears to provide the ability of changing keyboard layout with a keyboard shortcut.

Yes, of course (I use Ctrl+Menu-key). But, for example, if you have French, German, Spanish and English layouts, you might want to switch to each one with a specific shortcut and without cycling.

Please tell me if I'm missing something here.

**Comment by tsujan** [comment]
I labeled this with "enhancement" but there is a problem too:

Unless I'm missing something — in which case, it should be very hidden — there isn't even a menu to choose an *added* layout from. A left-click cycles through layouts and a right-click shows the settings menu.

**Comment by johnvranos** [comment]
Yes, you are right.

I am not one of the project developers, I just browsed the bug reports.

**Comment by tsujan** [comment]
The menu problem can be fixed easily, I guess, by adding an option to the widget settings for middle-clicking: a left-click could show the menu and a middle-click could change the layout.
